sdk/identity/azure-identity-broker/README.md

```diff
@@ -89,6 +89,16 @@ InteractiveBrowserCredential cred = new InteractiveBrowserBrokerCredentialBuilde
     .build();
 ```
 
+#### Use the default account for sign-in
+
+When this option is enabled, the credential will attempt to silently use the default broker account. If using the default account fails, the credential will fall back to interactive authentication.
```

Comment:
nit: Can add a one line description of what the default broker account here is.

Reply:
It turns out that's not actually true; on other platforms it may not be the user that is signed in. You could sign into another account in Intune.